### ISSUE TITLE
👹 Fera Formidável 4.3

### ISSUE THREAD

**Issue by ThallesCansi** [ThallesCansi/RNAG#13]
**Objetivo:** implemente o regularizador *dropout* na rede neural feita em Python puro.

**Comentário:** algumas tarefas vão apresentar palavras e conceitos que ainda não vimos em sala. Parte do desafio é justamente se informar sobre estes conceitos.

**Comentário 2:** observe que o enunciado diz claramente que é para realizar a tarefa na rede neural feita em Python puro nos vídeos da disciplina. Se você está usando o `PyTorch`, `numpy`, `tensorflow`, `keras`, `lightning` ou qualquer outra biblioteca pronta, você está no caminho errado!
